staking/programs/integrity-pool/src/lib.rs

```diff
@@ -48,6 +48,19 @@ pub mod integrity_pool {
         Ok(())
     }
 
+    pub fn update_delegation_fee(
+        ctx: Context<UpdateDelegationFee>,
+        delegation_fee: frac64,
+    ) -> Result<()> {
+        let pool_data = &mut ctx.accounts.pool_data.load_mut()?;
+
+        for publisher_index in 0..pool_data.delegation_fees.len() {
```

Comment:
`MAX_PUBLISHERS` instead of `pool_data.delegation_fees.len()`

Reply:
Fixed.